syscalls.go

```diff
@@ -469,26 +469,10 @@ var objNameAllowsDot = internal.FeatureTest("dot in object names", "5.2", func()
 	return true
 })
 
-func bpfGetMapFDByID(id uint32) (*internal.FD, error) {
-	// available from 4.13
-	attr := bpfGetFDByIDAttr{
-		id: id,
-	}
-	ptr, err := internal.BPF(_MapGetFDByID, unsafe.Pointer(&attr), unsafe.Sizeof(attr))
-	if err != nil {
-		return nil, xerrors.Errorf("can't get fd for map id %d: %w", id, err)
-	}
-	return internal.NewFD(uint32(ptr)), nil
-}
-
-func bpfGetProgramFDByID(id uint32) (*internal.FD, error) {
-	// available from 4.13
+func objGetFDByID(cmd int, fd uint32) (*internal.FD, error) {
```

Comment:
Agree with Tobias, also please keep the `bpf` prefix:

```suggestion
func bpfObjGetFDByID(cmd int, id uint32) (*internal.FD, error) {
```

Reply:
renamed it